code/modules/paperwork/paper.dm

```diff
@@ -484,3 +484,66 @@
 /obj/item/paper/evilfax/proc/evilpaper_selfdestruct()
 	visible_message("<span class='danger'>[src] spontaneously catches fire, and burns up!</span>")
 	qdel(src)
+
+/obj/item/paper/telecomms
+	name = "Your Guide To Telecomms!"
+	info = "<b>Congratulations On Your Purchase Of An NT-07 Telecommunications System!</b><br>\
+	WARNING: Should this machinery become destroyed or non functional, please contact your nearest technician.<br>\
+	INFO: This machinery is very subsceptable to Ionspheric Interference. Should an Ionspheric Anomalie occur, the machinery may be offline for a short amount of time.<br>\
```

Comment:
`subsceptable` -> `susceptible`

`Ionspheric` -> `ionospheric` and not capitalised `interference` either

`Ionspheric` -> `ionospheric` again. Also `Anomalie` -> `anomaly`

`short amount` -> `brief period`